pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -3501,48 +3501,62 @@ def prepare_plots(self):
         rdd = self.raw_data_dict
 
         for qb in self.qb_names:
+            for p, param_values in enumerate([self.amps, self.freqs]):
+                if param_values is None:
+                    continue
+                suffix = '_amp' if p == 0 else '_freq'
+                mask = pdd['mask'][qb]
+                xlabel = r'Flux pulse amplitude' if p == 0 else \
+                    r'Derived qubit frequency'
 
-            # Plot T1 vs flux pulse amplitude
-            mask = pdd['mask'][qb]
-            label = f'T1_fit_{qb}'
-            xvals = self.amps[qb][mask] if self.freqs[qb] is None else \
-                self.freqs[qb][mask]
-            xlabel = r'Flux pulse amplitude' if self.freqs[qb] is None else \
-                r'Derived qubit frequency'
-            self.plot_dicts[label] = {
-                'title': rdd['measurementstring'] + '\n' + rdd['timestamp'],
-                'plotfn': self.plot_line,
-                'linestyle': '-',
-                'xvals': xvals,
-                'yvals': pdd['T1'][qb][mask],
-                'yerr': pdd['T1_err'][qb][mask],
-                'xlabel': xlabel,
-                'xunit': 'V' if self.freqs[qb] is None else 'Hz',
-                'ylabel': r'T1',
-                'yunit': 's',
-                'color': 'blue',
```

Comment:
I've removed this setting of the color to 'blue' in order to have the same color for all plots.